targets/csharp/make.js

```diff
@@ -100,9 +66,9 @@ function getBaseTypeSyntax(datatype) {
 }
 
 function makeDatatypes(apis, sourceDir, apiOutputDir) {
-    var modelTemplate = getCompiledTemplate(path.resolve(sourceDir, "templates/Model.cs.ejs"));
-    var modelsTemplate = getCompiledTemplate(path.resolve(sourceDir, "templates/Models.cs.ejs"));
-    var enumTemplate = getCompiledTemplate(path.resolve(sourceDir, "templates/Enum.cs.ejs"));
+    var modelTemplate = getCompiledTemplate(path.resolve(sourceDir, "templates", "Model.cs.ejs"));
```

Comment:
While I applaud your sense of correctness, I think that here today, this is "changing a lot of lines for no purpose".
This is a big PR, and I'd rather not have no-op line changes for a PR that is otherwise already complicated.